pkg/systemd/logDetails.jsx

```diff
@@ -90,32 +107,32 @@ export class LogEntry extends React.Component {
 
         this.loadProblem = this.loadProblem.bind(this);
         this.goHome = this.goHome.bind(this);
+        this.problems_client = null;
     }
 
     loadProblem(entry) {
-        const problems_client = cockpit.dbus('org.freedesktop.problems', { superuser: "try" });
-        const service = problems_client.proxy('org.freedesktop.Problems2', '/org/freedesktop/Problems2');
-        const problems = problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
-
-        problems.wait(() => {
-            service.GetProblems(0, {})
-                    .then(problem_paths => {
-                        const fields = [entry.PROBLEM_DIR, entry.PROBLEM_DUPHASH, entry.PROBLEM_UUID];
-                        let path = null;
-                        problem_paths.some(pth => {
-                            const p = problems[pth];
-                            if (p && (fields.indexOf(p.ID) > 0 || fields.indexOf(p.UUID) || fields.indexOf(p.Duphash))) {
-                                path = p;
-                                return true;
-                            } else {
-                                return false;
-                            }
-                        });
-
-                        this.setState({ entry: entry, loading: false, error: "", problemPath: path, abrtService: service });
-                    })
-                    .catch(err => this.setState({ entry: entry, loading: false, error: err.toString() }));
-        });
+        if (this.problems_client)
+            this.problems_client.close();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18052-20221219-103616-9aa9eeeb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)